### ISSUE TITLE
Reproducing Replogle Nadig Dataset benchmarking (Figure 2 from STATE manuscript)

### ISSUE THREAD

**Issue by ktiwar92** [ArcInstitute/state#163]
Hi,

Thank you for creating such an interesting tool! 
We wanted to reproduce the benchmarking results for the Replogle Nadig genetic perturbation data. While some details (total cells, target genes, hyperparameters like cell set size etc) were available in the paper, we needed some more details around dataset pre-processing and train/test/val split:

1. Pre-processing:
- Was the K562 genome wide screen from Reploge also included in training ?
- For on target KD based filtering of these datasets, should we filter based on the cell-load default thresholds (pert level: 70% KD, cell level: 50% KD, min cells:30) ?
- Was each cell line dataset processed separately to identify 2000 HVGs or merged to identify a common set of 2000 HVGs across the datasets? 

2. Train/Test/Val Split:
- Could you please check, if the following understanding regarding train/test/val split of the data is correct:
- As we understand,
       - 4 Training runs were conducted, with 3 cell lines in train and 1 held out for testing
       - For each held out test cell line, cells corresponding to a 30% random sample of it's target genes were also added to train & remaining 70% target genes were split between Test and Validation sets? 
       - if this is correct, could you please provide more details on how you split the test and validation sets (ex. 90/10 split) ? 
- If this is incorrect could you please provide more details regarding how you split the data 
- if you still have the splits/config.toml files available for this benchmarking run, it would be great if you can share those !

Thank you!

**Comment by ktiwar92** [comment]
Thank you for sharing these files @abhinadduri !

I had one more question regarding the Train/Test split.. the publication mentions that: **For the Replogle-Nadig dataset, we conducted an evaluation by iteratively holding out one cell line as a test set. For each iteration, models were trained on the remaining three cell lines plus an additional 30% of perturbations randomly sampled from the test cell line.** ... 
We understood this to mean that 70% of the held out cell line perturbations are split b/w Test and Val and remaining 30% are used as part of the training data.. but based on the config files shared by you, it seems that it was the opposite case i.e. 35% of perturbations in Test & Val, and 70% in train ? Could you clarify if this is indeed the case, or unless if we are misunderstanding how the perturbations were split b/w train/test/val 

Thanks!
